src/pages/Stats/StatsComponents.jsx

```
@@ -1,979 +1,411 @@
-import ChevronDown from 'lucide-react/dist/esm/icons/chevron-down';
+import React, { useState, useMemo, useRef, useEffect } from 'react';
+import Search       from 'lucide-react/dist/esm/icons/search';
+import X            from 'lucide-react/dist/esm/icons/x';
+import Users        from 'lucide-react/dist/esm/icons/users';
+import User         from 'lucide-react/dist/esm/icons/user';
+import Trophy       from 'lucide-react/dist/esm/icons/trophy';
+import ChevronDown  from 'lucide-react/dist/esm/icons/chevron-down';
 import ArrowLeftRight from 'lucide-react/dist/esm/icons/arrow-left-right';
 
-import { STATS_COLORS as COLORS, MC_FONT, MOCK_PLAYERS, MOCK_TEAMS, PLAYERS_BY_NAME, formatNumber } from './statsUtils.js';
+import { MOCK_PLAYERS, MOCK_TEAMS, PLAYERS_BY_NAME, formatNumber } from './statsUtils.js';
```

ESLint

[error] 'formatNumber' is defined but never used. Allowed unused vars must match /^[A-Z_]/u.

(no-unused-vars)